skunkworks_crow/build.gradle

```diff
@@ -74,4 +74,5 @@ dependencies {
     // Permissions Dispatcher
     implementation "org.permissionsdispatcher:permissionsdispatcher:4.5.0"
     annotationProcessor "org.permissionsdispatcher:permissionsdispatcher-processor:4.5.0"
+    implementation 'androidx.browser:browser:1.2.0'
```

Comment:
Why introducing a third-party lib for this small feature @devanshi7799 ? If I were you, I will implement that using a simple web view. 

Downsides for introducing a new lib in this case are:
1. Some third-party libs have many redundant functions, that will increase the apk size. 
2. Some third-party libs are not well tested that may introduce bugs that cannot fix.

So we are trying not to use third-party libs unless we have to do that, I think this would be helpful for you.

Reply:
> Why introducing a third-party lib for this small feature @devanshi7799 ? If I were you, I will implement that using a simple web view.

Hi @huangyz0918, it's a custom tabs dependency, and nowadays every app has in-app browser feature which provides a look and feel of the browser within the app without actually moving out of the app, so I think its fine we even have this in ODK-Collect but not AndroidX dependency. I think @devanshi7799 this is fine no need to explore the webview as of now, if any feature ask comes in future then we can pick that up or if it we face any issues due to this then we can pick this up.